src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala

```diff
@@ -173,4 +173,11 @@ class BuiltinStringFunctionsTest extends AnyFlatSpec with Matchers with FeelInte
 
     eval(" string length(uuid()) ") should be(ValNumber(36))
   }
+
+  it should "return a string encoded as base64" in {
```

Comment:
:wrench: We need to group the test cases by name. By using the keyword `it`, it adds the test case to the previous group above. 

```suggestion
  "A encode base64() function" should "return a string encoded as base64" in {
```